go/teams/implicit.go

```diff
@@ -176,7 +176,7 @@ func lookupImplicitTeamAndConflicts(ctx context.Context, g *libkb.GlobalContext,
 	team, err = Load(ctx, g, keybase1.LoadTeamArg{
 		ID:          teamID,
 		Public:      impTeamName.IsPublic,
-		ForceRepoll: true,
+		ForceRepoll: false, // it's important to hit the cache here, since we call this once per chat send!
```

Comment:
This function drives public functions which I'd expect to return the latest answer. Does chat not have its own cache?

Reply:
Or could plumb forceRepoll through

Reply:
no, chat just spams this (cc @mmaxim).  I was thinking though it should be OK to get a slightly stale answer here...